Update docs (#864)
VirtusLab/git-machete-intellij-plugin: CONTRIBUTING.md

  containing the name of the param.
* Avoid running code outside of IDE-managed threads.
  Use either UI thread (for lightweight operations) or `Task.Backgroundable` (for heavyweight operations).

> * Use `...` after action name texts that result with a dialog (like `Push...` or `Slide In Branch Below...`).

Reviewer: ```suggestion
* Append `...` to the names of actions that open a dialog (like `Push...` or `Slide In Branch Below...`) before performing the actual task.
```

> * Toolbar name texts of a toolbar actions that refer a branch should indicate the branch under action with a word `Current`.

Reviewer: ```suggestion
* Toolbar name texts of toolbar actions that refer a branch should indicate the branch under action with a word `Current`.
```

>   On the other hand, context-menu actions text names should be kept short (no `This`/`Selected`).

Reviewer: ```suggestion
  On the other hand, **context-menu** actions text names should be kept short (no `This`/`Selected`).
```

> ## UI conventions

Reviewer: Pls move to this section (one guideline is already present)
